src/main/java/core/basesyntax/sevice/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.sevice.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {

```

Comment:
The class 'ReportGeneratorImpl' should be placed in the 'service.impl' package according to the project structure defined in the checklist.